thirdeye/thirdeye-pinot/src/main/java/org/apache/pinot/thirdeye/detection/health/DetectionHealth.java

```diff
@@ -142,6 +142,17 @@ public Builder addDetectionTaskStatus(TaskManager taskDAO, long limit) {
       return this;
     }
 
+    /**
+     * Add the detection task health status in the health report built by the builder. Do not return any task details
+     * @param taskDAO the task dao
+     * @return the builder
+     */
+    public Builder addDetectionTaskStatus(TaskManager taskDAO) {
+      this.taskDAO = taskDAO;
+      this.taskLimit = 0;
```

Comment:
Why init taskLimit here?

Reply:
This is the number of taskDTO returned in the health report. Primarily used for debugging purposes. By default set it to 0 means don't return any task details.